backend/flask_api.py

```diff
@@ -30,6 +29,74 @@
 ALLOWED_EXTENSIONS = {'txt', 'json', 'json5'}
 app.config['UPLOAD_FOLDER'] = ''
 app.config['MAX_CONTENT_PATH'] = 1000000  # 100.000 byte = 100kb
+app.config['MONGODB_DB'] = 'knexDB'
+app.config['MONGODB_HOST'] = 'mongodb'
+app.config['MONGODB_PORT'] = 27017
+
+es = Elasticsearch([{'host': 'elasticsearch', 'port': 9200}])
+
+client = MongoClient('mongodb:27017')
```

Comment:
`client = MongoClient('mongodb:27017')` should'nt the client get the connection by using the config?
```
+app.config['MONGODB_DB'] = 'knexDB'
+app.config['MONGODB_HOST'] = 'mongodb'
+app.config['MONGODB_PORT'] = 27017
```
if u are startig to use the config at this point we should use it as well